IDEAS/Buildings/Components/Interfaces/RectangularZoneTemplateInterface.mo

```diff
@@ -836,6 +842,11 @@ public
         extent={{-20,20},{20,-20}},
         rotation=180,
         origin={-2,60})));
+  Modelica.Thermal.HeatTransfer.Interfaces.HeatPort_b gainEmb[nGainEmb] if
+    bouTypFlo == IDEAS.Buildings.Components.Interfaces.BoundaryType.InternalWall or
+    bouTypFlo == IDEAS.Buildings.Components.Interfaces.BoundaryType.BoundaryWall
```

Comment:
how about `SlabOnGround`, `OuterWall`? They can also be used for tabs, in theory. E.g. for a roof/floor.

Reply:
`Outerwall` is not an option for the floor but `slabOnGround` is. I overlooked it. I have added it now.

Reply:
`OuterWall` can be used when there are TABS in the roof?

Reply:
@damienpicard how about `OuterWall`?

Reply:
`Outerwall` is only possible for ceiling. As it would be confusing to have 2 embedded heat ports (one for floor, one for ceiling), I chose to only include one for floor.